tests/testsuite.at

```diff
@@ -208,7 +208,9 @@ dnl Defines all possible Souffle flag configurations for testing.
 dnl NOTE: This is the default configuration that can be overridden
 dnl using SOUFFLE_CONFS environment variable.
 m4_define([DEFAULT_CONFS], [[], dnl interpreter
+ [-t],
```

Comment:
Do we need to run all the tests with -t or is it enough to test with a new single/few tests using
.pragma "provenance" ""
(I ask this not knowing if .pragma can do this ...)